tests/sles4sap/sapconf.pm

```diff
@@ -83,28 +83,35 @@ sub run_developers_tests {
     type_string "cd\n";
 }
 
+sub verify_sapconf_service {
+    my ($svc, $desc) = @_;
+
+    my $output      = script_output "systemctl status $svc";
+    my $statusregex = $svc . ' - ' . $desc . '.+' . 'Loaded: loaded \(/usr/lib/systemd/system/' . $svc . ';.+';
+    my $active      = $statusregex . 'Active: active \((listening|running)\).+';
+    my $success     = $statusregex . 'Active: active \(exited\).+' . 'status=0\/SUCCESS';
+    die "Command 'systemctl status $svc' output is not recognized" unless ($output =~ m|$active|s or $output =~ m|$success|s);
+}
+
 sub run {
     my ($self) = @_;
 
     select_console 'root-console';
 
-    my $output = script_output "systemctl status tuned";
-    my $statusregex
-      = 'tuned.service - Dynamic System Tuning Daemon.+'
-      . 'Loaded: loaded \(/usr/lib/systemd/system/tuned.service;.+'
-      . 'Active: active \(running\).+'
-      . 'Starting Dynamic System Tuning Daemon.+'
-      . 'Started Dynamic System Tuning Daemon.';
-    die "Command 'systemctl status tuned' output is not recognized" unless ($output =~ m|$statusregex|s);
-
     assert_script_run("rpm -q sapconf");
 
-    $output = script_output "tuned-adm active";
+    verify_sapconf_service('tuned.service',   'Dynamic System Tuning Daemon');
+    verify_sapconf_service('sapconf.service', 'sapconf');
+    verify_sapconf_service('uuidd.socket',    'UUID daemon activation socket');
+    verify_sapconf_service('sysstat.service', 'Write information about system start to sysstat log')
+      if (sle_version_at_least('15'));
```

Comment:
As now we have nice is_sle implemenation, sle_version_at_least is deprecated. So, could you please replace it with is_sle('15+') here?

Reply:
Sure. But will do the same in the rest of the file. That's not the only use of `sle_version_at_least` there.